submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>papap</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <nav>
+        <ul class="header-icons">
+
+            <li class="greatest">
+                <label for="sq">
+                    <img class="h-image" src="icons/popup-button.png" alt="button">
+                </label>
+                <input tabindex="1" id="sq" type="checkbox" class="visually-hidden" />
+
+                <ul class="menu">
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/apps.png" alt="apps">
+                            <span>Apps</span></a></li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/blogs.png" alt="blogs">
+                            <span>Blogs</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/calendar.png"
+                                alt="calendar"><span>Calendar</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/chat.png" alt="chat">
+                            <span>Chat</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/classroom.png"
+                                alt="classroom"><span>Classroom</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/collection.png"
+                                alt="collection"><span>Collection</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/contacts.png"
+                                alt="contacts"><span>Contacts</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/documents.png"
```

Comment:
Fix the focused border(outline) on this icon all lines must be straight)
![image](https://user-images.githubusercontent.com/16196199/183469024-42638597-5eaf-4407-826e-013e43145bdb.png)